tests/test_feature_selector.py

```diff
@@ -167,6 +167,9 @@ def test_get_selected_features(fs, mosquito):
     X = mosquito.X[:, 0:10]
     y = np.array([1] + [0, 1] * 14)
     fs.fit(X, y)
+    fs._selected_features = SelectedFeatures(
+        min_feats=[0], max_feats=[0], mid_feats=[0]
+    )
```

Comment:
why did you need to add this? Will break once merged with target branch too due to renaming

Reply:
right, just to control which are the selected features, this way it's easier to test

Reply:
I'll rewrite it to be consistent with the naming (and merge the target branch in to run the tests)